frameworks/g-cloud-9/search_mapping.json

```diff
@@ -1,4 +1,27 @@
 {
+  "settings": {
+    "index": {
+      "number_of_shards": 1
```

Comment:
I'm going to take out the sharding change and stage that as a separate PR. Probably should have done that originally. So this PR will only be for stemming. :)